src/collective/easyform/permissions.zcml

```diff
@@ -2,73 +2,41 @@
   <permission
       id="collective.easyform.AddContent"
       title="collective.easyform: Add Content"
-      >
-    <role name="Manager" />
```

Comment:
Was this working at all? I never saw something like this in zcml!

Reply:
I also never saw this, I had to look twice and looked it up here: https://github.com/zopefoundation/AccessControl/blob/master/src/AccessControl/meta.zcml#L44 - yes it works.
But when adding it to the profile it's much easier to override.